pages/projects/coffea-casa.md

```diff
@@ -68,22 +75,33 @@ More information could be found in the corresponding repository:
 
 **Recent accomplishments**:
-- Deployed at University Nebraska-Lincoln Tier3, `coffea-casa` facility is ready to accommodate the first CMS users: [try it!](https://cmsaf-jh.unl.edu/)
+- Deployed at University Nebraska-Lincoln Tier2, `coffea-casa` CMS facility is accommodating the first CMS users: [try it!](https://coffea.casa/). Currently more then 140 users had been used the CMS facility over last year.
 
-![Coffea-casa Jupyterlab interface with Dask Labextention powered cluster](/assets/images/coffea-casa-1.png){:style="display:block; margin-left: auto; margin-right: auto; width: 50%"}
+![CMS Coffea-Casa Jupyterlab interface with Dask Labextention powered cluster](/assets/images/coffea-casa-3.png){:style="display:block; margin-left: auto; margin-right: auto; width: 50%"}
 
+- For non-CMS users, we enabled the Opendata `coffea-casa` facility: [try it!](https://coffea-opendata.casa/). Currently more then 60 users had been used the Opendata facility over last year.
 
-**Future plans for 2021**:
-- Release Helm charts and other by-products to be deployable on the other facilities
-- Deploy coffea-casa functionality at least on one external facility
-- Involve more physics analysis groups to use facility.
+![Opendata Coffea-Casa login web-page enabling a user access to Jupyterlab interface with Dask Labextention powered cluster](/assets/images/coffea-casa-3.png){:style="display:block; margin-left: auto; margin-right: auto; width: 50%"}
+
+- For ATLAS physists, there were deployed ATLAS Coffea-Casa analysis facility instance at University of Chicago.
 
+- Coffea-casa analysis facility is a key component for ![IRIS-HEP Analysis Grand Challenge](https://iris-hep.org/projects/agc.html) preparations.
+
+- Both Opendata `coffea-casa` analysis facility at University Nebraska-Lincoln and ATLAS analysis facility instance at University of Chicago were used for showcasting various Python analysis packages and services on ![Analysis Grand Challenge Tools workshop 2021](https://indico.cern.ch/event/1076231/) and ![Analysis Grand Challenge Tools workshop 2022](https://indico.cern.ch/event/1126109/) workshops.
+
+
+**Future plans for 2022**:
+- Test Helm charts and other by-products on the other facilities.
+- Involve more physics analysis groups to use facility.
+- Benchmark various software components and packages deployed at Coffea-Casa analysis facility at University Nebraska-Lincoln.
+- Prepare and execute Analysis Grand Challenge at Coffea-Casa Analysis Facility deployed at University Nebraska-Lincoln and University of Chicago.
 
 
 ### Recent videos and tutorials
 
-* The coffea-casa introductory Youtube video at [PyHEP 2020](https://www.youtube.com/watch?v=CDIFd1gDbSc)
-* The coffea-casa Youtube video tutorial at [PyHEP 2020](https://www.youtube.com/watch?v=oPl0t8J36-Q)
+*  The Coffea-Casa analysis facility demo "Scale-out with coffea: coffea-casa" - Youtube video at ![Analysis Grand Challenge Tools workshop 2021](https://youtu.be/7s30NNsAG7w)
```

Comment:
for future, you can include these in _data/videos and they will appear on your page (if its a focus-area layout anyway)